.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: ruby
 before_install:
-  - rm ./Gemfile.lock
+  - rm ./gemfiles/Gemfile.*.lock
```

Comment:
The CI step that removes lockfiles is confusing to me. I'm not sure why we even check in a lockfile if we don't use it in CI. @Edouard-chin I believe you introduced this two years ago. Do you still think this is a good idea?

I'd rather have something like dependabot bump our dependencies and always use the same lockfile.

Reply:
I can't recall a single other project I contribute to that checks in the lockfiles, which is why I opened #175

Reply:
I agree. I'll remove the lockfiles in this PR.

Reply:
Yeah, I think when you're deploying an application, it can make sense to have a lockfile floating around. We're not deploying/publishing a binary here, so it's likely providing little to no value.

Reply:
Few years ago the ruby community shifted to include the Gemfile.lock under VCS. It makes development of a library as well as bisecting much easier. Bundler explained it in more details in a blog https://bundler.io/blog/2018/01/17/making-gem-development-a-little-better.html

> The CI step that removes lockfiles is confusing to me. 

When CI runs we want to make sure that the library works on the most recent version of the gem. Though now that dependabot is a thing, I'm fine to use the same lockfile

Reply:
> I agree. I'll remove the lockfiles in this PR.

Please put it back. Almost all shopify projects have the Gemfile.lock commited the lockfile and the ones that didn't should be updated. When bootrapping a gem nowaway, the Gemfile.lock isn't gitignored

Reply:
Why does it make sense to have a Gemfile.lock checked in that is not tested?

🤔 

I also really don't want to get caught up in the arguments here... I'm trying to solve a different problem. I think we should leave the discussion about what to do with the lockfile in CI for @marcandre's issue https://github.com/Shopify/erb-lint/issues/175.

For this PR, I'll keep the semantics that currently exist on master.

I'll keep `Gemfile.lock` in place so that it will be used by default on developer's computers, and add my two separate gemfiles for CI - I just wouldn't check in the lockfiles for these CI-specific gemfiles, and instead ignore them.

Reply:
>  I just wouldn't check in the lockfiles for these CI-specific gemfiles, and instead ignore them.

Yes that's the right thing to do